pkg/operator/apis/aro.openshift.io/v1alpha1/cluster_types.go

```diff
@@ -48,6 +48,30 @@ type InternetCheckerSpec struct {
 	URLs []string `json:"urls,omitempty"`
 }
 
+type OperatorFlags map[string]string
+
+func (f OperatorFlags) GetBoolean(key string) (val bool, exists bool) {
+	v, ext := f[key]
+	if ext {
+		// Only accept a literal true, rather than accepting anything other than
+		// literal false as true
+		// TODO: Is this the best behaviour?
+		if v == "true" {
```

Comment:
```suggestion
		if strings.EqualFold(v,"true") {
```
to make it utf-8 case insensitive.

Reply:
:+1: